### ISSUE TITLE
links in twisted.words docs are broken

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4954]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)| @habnabit reported|
|-|-|
|Trac ID|trac#4954|
|Type|defect|
|Created|2011-03-15 05:39:26Z|

Spidered the most recent docs; all of these links are broken in the documentation. Format is the document with broken links, followed by the broken links.

```
http://twistedmatrix.com/documents/current/words/howto/im.html
  http://twistedmatrix.com/documents/10.2.0/api/twisted.im.baseaccount.AccountManager.html
  http://twistedmatrix.com/documents/10.2.0/api/twisted.im.basechat.ChatUI.html
  http://twistedmatrix.com/documents/10.2.0/api/twisted.im.basechat.Conversation.html
```

<details><summary>Searchable metadata</summary>

```
trac-id__4954 4954
type__defect defect
reporter__habnabit habnabit
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__invalid invalid
component__words words
keywords__documentation documentation
time__1300167566000000 1300167566000000
changetime__1300582436000000 1300582436000000
version__None None
owner__ 
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_2"></a>|@habnabit commented|
|-|-|

This seems to be fixed in [#4054](https://github.com/twisted/twisted/issues/4054)? Maybe the documentation just hasn't been rebuilt since then. It's still in the live pages, though.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh set status to `closed`|
|-|-|

Cannot reproduce in [current buildbot sphinx docs](http://buildbot.twistedmatrix.com/builds/sphinx-html/264-15833/projects/words/howto/im.html).
